app/templates/FeedPage.html

```diff
@@ -1,50 +1,73 @@
 {% extends "layout.html" %}
 {% block content %}
 
+<br>
+<br>
+
+<!-- creating button redirecting user to create job post page-->
+<button class = 'btn btn-info btn-sm' onclick="location.href='{{ url_for('JobPost') }}'">
+    <!-- adding pen picture to button label-->
+    <span class= "fa fa-pen"></span> Create a Job Post
+</button>
```

Comment:
This should be in a container similar to the previous code. To resolve the merge conflict, leave the old code and add changes individually in a future pull request.